sympy/matrices/dense.py

```diff
@@ -53,6 +45,9 @@ def __eq__(self, other):
         except AttributeError:
             return False
 
+    def __ne__(self, other):
```

Comment:
Maybe this is not necessary (https://docs.python.org/3/reference/datamodel.html#object.__ne__):

    By default, __ne__() delegates to __eq__() and inverts the result unless it is NotImplemented.